src/main/java/com/faforever/client/vault/VaultEntityController.java

```diff
@@ -131,9 +131,6 @@ public void initialize() {
 
     initSearchController();
 
-    BooleanBinding inSearchableState = Bindings.createBooleanBinding(() -> state.get() != State.SEARCHING, state);
-    searchController.setSearchButtonDisabledCondition(inSearchableState);
-
```

Comment:
We should still have the button disabled when searching sorry if that wasn't clear